h/static/scripts/directive/annotation.js

```diff
@@ -454,6 +454,10 @@ function AnnotationController(
     return vm.annotation.target;
   };
 
+  vm.orphan = function() {
```

Comment:
For consistency with the other code here and also for clarity, I would suggest that functions returning booleans should generally have names beginning with `is` or `has`